webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/components/General.js

```diff
@@ -111,6 +122,7 @@ General.defaultProps = {
   operatingSystemId: undefined,
   operatingSystemTemplate: undefined,
   smartProxyId: undefined,
+  downloadUtility: undefined,
```

Comment:
If it's semi required and actually expects one of the supported utils, maybe it should be `curl` by default instead?